.github/workflows/pages.yaml

```
@@ -92,6 +92,12 @@ jobs:
 
       - run: python3 -m pip install -r master/scripts/requirements.txt
 
+      - name: Generate dependency graph rendering
+        run:
+          python3 master/scripts/generate_dependency_graph_rendering.py
```

Could this be a Makefile rule? It's best to have as little bespoke logic in CI pipelines as possible.

---

sure, I can do that. Though, I don't know how to "silently fail" the script in the makefile, and I don't want our CI to fail if this script breaks at some point

---

figured it out I think